server/models/Sitter.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require('mongoose');
+const profileSechma = require('./Profile');
+
+const Profile = mongoose.model(Profile, profileSechma);
+const options = { discriminatorKey: 'kind' };
+
+const Sitter = Profile.discriminator(
+  'Sitter',
+  new mongoose.Schema({
```

Comment:
We should base the branch of this PR off of `sitterSchema` so this file isn't showing but that PR is approved so just merge the sitter schema PR in and this file will go away 🙂

Reply:
I branched of that just because I needed to import the sitter model